handlers/tokeninfo/proxy/handler.go

```diff
@@ -0,0 +1,25 @@
+package tokeninfoproxy
+
+import (
+	"net/http"
+	"net/http/httputil"
+	"net/url"
+	"github.com/zalando/planb-tokeninfo/handlers/tokeninfo"
+)
+
+type tokenInfoProxyHandler struct {
+	upstream *httputil.ReverseProxy
+}
+
+func (h *tokenInfoProxyHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
+	token := req.URL.Query().Get("access_token")
```

Comment:
Sounds legit, I saw that from your previous changes. I'll add them later. Can I have a single issue for that? :)